IREC/IREC_PDD.tex

```diff
@@ -0,0 +1,288 @@
+in this event provides room to grow academically and professionally for every student involved.
+
+%\begin{figure}
+  %\includegraphics[width=\linewidth]{figs/project-life-cycle.png}
+  %\caption{A PDD is the first piece of documentation to be archived in the project life cycle. Since the life cycle can be iterative, a new design document may also refer to one or more previous SPPs.}
+%\label{fig:lifecycle}
+%\end{figure}
+
+% \section{Secondary Objectives}
+% \label{sec:secondary-obj}
+% Secondary Objectives are lower priority or bonus objectives that are significant but not the main focus of the project. This template does not have secondary objectives.
+
+\section{Benefit to SPEX}
+\label{sec:benefit}
+% One of the core values of SPEX is to provide opportunities for academic and professional growth for its members,
+% and to challenge them with interesting projects.
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+SPEX's involvement in this globally-identified compeition will serve to provide exposure and merit to the team as a whole.
+Individual students engaged in this endeavor will also benefit by improving technical knowledge and abilities while
+participating in a major event which can be used to improve their resumes.
+
+% Below I have used subsections to identify key ideas in this section. These particular subsections are not required as part of the SPEX Standard, but serve as an example of using subsections in a text
+
+\subsection{Benefit to RIT}
+\label{subsec:Benefit to RIT}
+By attending the event and competing against top universities, RIT will be able to use this as an opportunity to showcase
+how passionate and involved their students are in space exploration. This involvement can be used in media as a tool for
+achieving more funds and attracting new students to the University. The media that results from the competition will provide
+exposure to SPEX and can be used when negotiating with the university when support is required like obtaining more work space or
+additional tools and equipment.
+
+\subsection{Partnership with Launch Initiative}
+\label{subsec:Partnership with Launch Initiative}
+Since the founding of SPEX, there has never been a strong relationship with the Launch Initiative team. By cooperatively participating
+in this event, a great opportunity is presented to create a good relationship between two of the largest space-related teams on
+campus. In the future, united efforts between the two teams could result in more advanced projects which will provide
+even more exposure to the RIT space teams.
+
+\section{Implementation}
+\label{sec:Implementation}
+  % What path do you anticipate the project to take?
+Due to a few meetings which occurred over the summer and at the beginning of the semester with students from both SPEX and LI
```

Comment:
I am not sure if talking about current progress or past work on the project is appropriate for the scope and purpose of this document.

Reply:
@Beaudidly is correct. PDD should be independent of the time it was written. The purpose is to capture the starting point (the kernel?) of a project and its objectives, and to make sure all parts of it are considered -- from funding to faculty support to the payload. Not everything needs to be _defined_ as long as goals and potential risks are _considered_. Think traceability.